package.json

```
@@ -4,7 +4,8 @@
   "description": "",
   "scripts": {
     "start": "func start",
-    "test": "jest"
+    "test": "jest",
+    "init-db": "psql -d mapping_viz_local < ./db/schema.sql && psql -d mapping_viz_local < ./db/data/test_data.sql "
```

you probably want to use / respect the env vars here to set the pg database to modify. I'd probably extract this to an executable shell script with some logging e.g.: 1. initializing the schema
2. loading test data etc
```suggestion
    "init-db": "psql -d $PGDATABASE < ./db/schema.sql && psql -d $PGDATABASE < ./db/data/test_data.sql"
```
https://www.postgresql.org/docs/9.5/libpq-envars.html

---

Good idea with using an env var for the db name here 👍